src/amo/components/AddonReviewManager/index.js

```diff
@@ -88,41 +88,59 @@ export class AddonReviewManagerBase extends React.Component<InternalProps> {
     );
 
     /* eslint-disable react/no-danger */
-    const formFooter = (
+    const formFooter = !review.isDeveloperReply ? (
```

Comment:
is worth setting review.isDeveloperReply to a var since it's used a few times here?